### PR TITLE
Google Optimize + First experiment working

### PR DESCRIPTION
**Pull request by marcoacfilho**

### CODE REVIEW

File: packages/client/src/comp/util/Signature.tsx [hunk 1]
```diff
@@ -18,11 +20,13 @@ type Props = React.PropsWithChildren<{
...
 export const Signature: React.FC<Props> = ({ setSignature }) => {
+  const { search } = useLocation()
+  const params = parseQS(search)
   return <Margin>
-    <Switchable>
+    <Switchable visible={Boolean(!params.case)}>
     {
       (choice: Choice) => {
-        if (choice === 'canvas') {
+        if (choice === 'canvas' || params.case === 'a') {
```
Comment (tianhuil) [inline packages/client/src/comp/util/Signature.tsx:29]: Can we name the test cases better?  Maybe they can be 'both' or 'upload'?  We are testing whether they can be both Canvas and Upload or just Upload.  I thin you have the logic wrong here.

File: packages/client/src/comp/util/Signature.tsx [hunk 2]
```diff
@@ -18,11 +20,13 @@ type Props = React.PropsWithChildren<{
 }>
 
 export const Signature: React.FC<Props> = ({ setSignature }) => {
+  const { search } = useLocation()
```
Comment (tianhuil) [inline packages/client/src/comp/util/Signature.tsx:23]: We actually return `query` in `useAppHistory`, which is what I think you want to actually use.

File: packages/client/src/comp/util/Signature.tsx (hunk 1 above)
Comment (marcoacfilho) [reply packages/client/src/comp/util/Signature.tsx]: done

File: packages/client/src/comp/util/Signature.tsx (hunk 2 above)
Comment (marcoacfilho) [reply packages/client/src/comp/util/Signature.tsx]: done

**Pull request merged**